index.js

```diff
@@ -85,20 +77,23 @@ FeatureService.prototype.request = function (url, callback) {
   // to ensure things are encoded just right for ArcGIS
   var encoded = encodeURI(decodeURI(url))
   var options = {
-    method: 'GET',
-    url: encoded
+    timeout: this.options.timeOut,
+    headers: {
+      'user-agent': 'Featureservices-Node'
+    }
+
   }
-  this._request(options, function (err, res) {
+  this._request(encoded, options, function (err, data, res) {
     if (err) {
-      if (err.message === 'ESOCKETTIMEDOUT') err.code = 504
+      if (err.code === 'ESOCKETTIMEDOUT') err.code = 504
```

Comment:
why is this changed? There are some pretty specific needs upstream in Koop that depend on the error structure.